src/changes-backwards-compatible.apib

```diff
@@ -11,6 +11,8 @@ We list all backwards-compatible additions here. These are currently available i
 - We added a new sorting option `weighted_value` to `deals.list` and will allow ascending as well as descending orders on sorts.
   The default remains created_at and descending as before if nothing is passed.
 
+- We will return the `weighted_value` in the response of both deals.list and deals.info requests.
```

Comment:
PP: it could make a lot of sense to rewrite the changes for `deals.*` into a single item with subitems, to keep the changes on those endpoints uncluttered.

Reply:
I think its clear enough so I will skip this.